src/llms/adapter-factory.ts

```
@@ -18,6 +18,10 @@ export class LLMAdapterFactory {
             `Gemini adapter failed to load: ${error}. Install @google/generative-ai to use Gemini: bun add @google/generative-ai`,
           );
         }
+      case "openai":
+      case "openai-compatible":
+        const { OpenAILLMAdapter } = await import("./openai.js");
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)